src/ifvc.c

```diff
@@ -201,8 +201,10 @@ void rebuildIfVc () {
 **
 */
 void buildIfVc(void) {
-    struct ifreq IfVc[ sizeof( IfDescVc ) / sizeof( IfDescVc[ 0 ] )  ];
+    struct ifreq IfVc[ MAX_IF * 2 ]; /* allocate in double in order to get as much information as possible */
```

Comment:
Why `*2`? Maximal number is `maximal`. So either defined constant is not so large or there is another hidden problem?